x/ccv/provider/keeper/keeper.go

```diff
@@ -887,3 +897,21 @@ func (k Keeper) GetFirstVscSendTimestamp(ctx sdk.Context, chainID string) (vscSe
 
 	return types.VscSendTimestamp{}, false
 }
+
+// SetLastDowntimeValsetUpdateId sets the latest valsetUpdateId when consumer initiated downtime slashing occured on chainID.
```

Comment:
Delete this state in `StopConsumer`. Also add logic for export/import.

Reply:
Done